lib/filter.g

```diff
@@ -184,6 +184,64 @@ BIND_GLOBAL( "InstallTrueMethodNewFilter", function ( tofilt, from )
 end );
 
 
+#############################################################################
+##
+#F  SuspendMethodReordering( )
+#F  ResumeMethodReordering( )
+#F  ResetMethodReordering( )
+##
+##  <#GAPDoc Label="MethodReordering">
+##  <ManSection>
+##  <Func Name="SuspendMethodReordering" Arg=""/>
+##  <Func Name="ResumeMethodReordering" Arg=""/>
+##  <Func Name="ResetMethodReordering" Arg=""/>
+##
+##  <Description>
+##  These functions control whether the method reordering process described
+##  in <Ref Func="InstallTrueMethod"/> is invoked or not. Since this process
+##  can be comparatively time-consuming, it is usually suspended when a lot
+##  of implications are due to be installed, for instance when loading the
+##  library, or a package. This is done by calling
+##  <C>SuspendMethodReordering()</C> once the installations are done,
+##  <C>ResumeMethodReordering()</C> should be called. These pairs of calls
+##  can be nested. When the outermost pair is complete, method reordering
+##  takes place and is enabled in <C>InstallTrueMethod</C> thereafter.
```

Comment:
Better write `<Ref Func="InstallTrueMethod"/>`.

Reply:
Agreed; and the same for `SuspendMethodReordering` etc. -- all changed now.